hugo/content/exporter/_index.md

```diff
@@ -259,7 +259,7 @@ sudo systemctl status crunchy-postgres-exporter@postgres_exporter_pg##_per_db
 
 ### Monitoring multiple databases and/or running multiple postgres exporters (RHEL / CentOS 7)
 
-Certain metrics are not cluster-wide, so in that case multiple exporters must be run to collect all relevant metrics. As of v0.5.x of postgres_exporter, a single service can connect to multiple databases, so as long as you're using the same custom query file for all of them, only one additional exporter service is required and this comes with pgmonitor 4.0 and above by default. The queries_per_db.yml file contains these queries and the secondary exporter can use this file to collect those metrics and avoid duplicating cluster-wide metrics. Note that some other metrics are per database as well (bloat). You can then define multiple targets for that one job in Prometheus so that all the metrics are collected together for a single database instance. Note that the "setup_*.sql" file does not need to be run on these additional databases if using the queries that pgmonitor comes with.
+Certain metrics are not cluster-wide, so in that case multiple exporters must be run to collect all relevant metrics. As of v0.5.x of postgres_exporter, a single service can connect to multiple databases. As long as you're using the same custom query file for all of those databases, only one additional exporter service is required and this comes with pgmonitor 4.0 and above by default. The queries_per_db.yml file contains these queries and the secondary exporter can use this file to collect those metrics and avoid duplicating cluster-wide metrics. Note that some other metrics are per database as well (bloat). You can then define multiple targets for that one job in Prometheus so that all the metrics are collected together for a single database instance. Note that the "setup_*.sql" file does not need to be run on these additional databases if using the queries that pgmonitor comes with.
```

Comment:
~in that case~

Note that the bloat metrics are per database as well; you can define multiple targets for the bloat job in Prometheus so that all the metrics are collected together for a single database instance.

pgMonitor

Reply:
> Note that the bloat metrics are per database as well; you can define multiple targets for the bloat job in Prometheus so that all the metrics are collected together for a single database instance.
> 

I kind of wanted to leave this section a bit more ambiguous since there may be more per-db metrics coming. Bloat is just the current example, so clarified it as such. And that's not really accurate that you're defining multiple targets for the bloat job in prometheus. You're defining multiple targets for the host job itself (multiple IPs/hostnames in the target definition).

Look good now?

Reply:
yup